src/main/java/org/opentripplanner/raptor/api/request/MultiCriteriaRequest.java

```diff
@@ -76,7 +77,7 @@ public Optional<RaptorTransitPriorityGroupCalculator> transitPriorityCalculator(
     return Optional.ofNullable(transitPriorityCalculator);
   }
 
-  public Optional<PassThroughPoints> passThroughPoints() {
+  public Optional<List<PassThroughPoint>> passThroughPoints() {
```

Comment:
What is the difference between an empty Optional and an Optional with an empty list inside? Isn't a list already a kind of optional type?

Reply:
Your are right - I saw it, but had to fix something else first and then forgot to fix that one. Thanks!

Reply:
When I wrote the original code I discovered that some parts of the code - no surprise! - depends on if pass-through points are configured or not, and that it's important to propagate this fact. We can of course agree on that an empty list of pass-through points signifies this, but it's important to be explicit.